Tools/Scripts/run-mvt-tests

```diff
@@ -0,0 +1,305 @@
+                "WEBKIT_INJECTED_BUNDLE_PATH": "{}/WebKitBuild/{}/lib".format(top_level_directory, configuration),
+                "COG_MODULEDIR": "{}/WebKitBuild/{}/Tools/cog-prefix/src/cog-build/platform".format(top_level_directory, configuration),
+            }
+            cmd = [os.path.join(top_level_directory, "Tools/jhbuild/jhbuild-wrapper"),'--wpe','--{}'.format(self.configuration),'run','/bin/bash','-c']
+            self._run(cmd, env)
+        else:
+            # TODO: Run with system libraries.
+            sys.stderr.writeln("Run on system libraries")
+            pass
+
+    def stop(self):
+        try:
+            command = "pkill {}".format(os.path.basename(self.binary))
+            subprocess.run(command.split(" "))
+        except CalledProcessError:
+            return False
+        return True
+
+    def _isRunning(self):
+        try:
+            command = "pgrep {}".format(os.path.basename(self.binary))
+            subprocess.check_call(command.split(" "), stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        except CalledProcessError:
+            return False
+        return True
+
+    def _isFlatpakSDK(self):
+        return not os.getenv("WEBKIT_JHBUILD") or os.getenv("WEBKIT_JHBUILD") == 0
+
+    def _isJHBuild(self):
+        return os.getenv("WEBKIT_JHBUILD") and os.getenv("WEBKIT_JHBUILD") == 1
+
+    def _run(self, cmd, env):
+        def dict_flat(d):
+            return " ".join([f"{key}={value}" for key, value in d.items()])
+        try:
+            env = {
+                "WEBKIT_EXEC_PATH": "/app/webkit/WebKitBuild/{}/bin".format(self.configuration.capitalize()),
+                "WEBKIT_INJECTED_BUNDLE_PATH": "/app/webkit/WebKitBuild/{}/lib".format(self.configuration.capitalize()),
+                "COG_MODULEDIR": "/app/webkit/WebKitBuild/{}/Tools/cog-prefix/src/cog-build/platform".format(self.configuration.capitalize()),
+            }
+            subcommand = "{} {} --host=all --port={}".format(dict_flat(env), self.binary, self.port)
+            cmd.append(subcommand)
+            subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        except CalledProcessError:
+            sys.exit(1)
+        time.sleep(2)
+
+
+def main():
+    """Script entry point."""
+
+    args = parse_args()
+
+    # 1. Start WPEWebDriver.
+    print("Start WPEWebDriver:", end=' ')
+    wpeWebDriver = WPEWebDriver(args)
+    wpeWebDriver.start()
+    print("OK")
+        
+    # 2. Start the WebDriver
+    args.browser = os.path.join("/home/dpino/workspace/gh/webkit", "WebKitBuild/WPE/Release/Tools/cog-prefix/src/cog-build/launcher/cog")
+    print("Connecting to remote driver:", end=' ')
+    driver = webdriver.Remote(
+        command_executor=args.address,
+        options=options(args.browser, args.platform),
+    )
+    print("OK")
+
+    server = os.getenv("MVT_INSTANCE_ADDRESS", "https://mvt.rdkcentral.com")
```

Comment:
why we do pass this via the environment instead of using an argument like the other parameters?